src/main/flight/pid.c

```diff
@@ -933,6 +948,24 @@ void checkItermLimitingActive(pidState_t *pidState)
     pidState->itermLimitActive = STATE(ANTI_WINDUP) || shouldActivate; 
 }
 
+void checkItermFreezingActive(pidState_t *pidState, flight_dynamics_index_t axis)
+{
+    if (usedPidControllerType == PID_TYPE_PIFF && pidProfile()->fixedWingYawItermBankLimit != 0) {
+        // Do not allow yaw I-term to grow when bank angle is too large
+        float bankAngle = DECIDEGREES_TO_DEGREES(attitude.values.roll);
+        if (axis == FD_YAW && fabsf(bankAngle) > pidProfile()->fixedWingYawItermBankLimit && !(FLIGHT_MODE(AUTO_TUNE) || FLIGHT_MODE(TURN_ASSISTANT) || navigationRequiresTurnAssistance())){
```

Comment:
`axis == FD_YAW` check can be done in the top if, that would be more readable I think

Reply:
Would there be a performance difference? There are basically five things that need to be checked: PIFF controller, feature enabled, axis, bank angle, flight modes. My reasoning was to affect multirotor and fixed wing without this feature as little as possible by nesting everything in that if statement.